matrix-sdk-android/src/main/java/im/vector/matrix/android/api/pushrules/PushRuleService.kt

```diff
@@ -42,6 +42,7 @@ interface PushRuleService {
 
     interface PushRuleListener {
         fun onMatchRule(event: Event, actions: List<Action>)
+        fun onRoomJoined(roomId: String)
         fun onRoomLeft(roomId: String)
         fun onEventRedacted(redactedEventId: String)
         fun batchFinish()
```

Comment:
This class is now handling more things that events that matched a push rules, could be renamed to something else? NoteworthyEventListener? ImportantEvents? or something more related to the fact that it will be used by the notification drawer? NotificationActionsListener?

Reply:
Indeed, you are right, we will rename that later